olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/DataRequest.h

```diff
@@ -53,47 +71,27 @@ class DATASERVICE_READ_API DataRequest final {
    * @brief WithPartitionId sets the request's Partition Id. If the partition
    * cannot be found in the layer, the callback will come back with an empty
    * response (null for data and error)
-   * @param partitionId the Partition Id.
+   * @param partition_id the Partition Id.
    * @return a reference to the updated DataRequest.
    */
   inline DataRequest& WithPartitionId(
-      boost::optional<std::string> partitionId) {
-    partition_id_ = partitionId;
+      boost::optional<std::string> partition_id) {
+    partition_id_ = partition_id;
```

Comment:
```suggestion
    partition_id_ = std::move(partition_id);
```
No need to invoke copy operator, cause we can use move-assignment operator here.